.github/workflows/unit-test.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
       name: Install dev dependencies
+    - run: npm run format
```

Comment:
why this is needed?

Reply:
You would need it if you want to make sure all files geing merged into the master branch to be formatted since there are some ways to create a commit skipping husky. Let's delete it if you think it's too much.

Reply:
If one wants to skip then let them skip. There is no need to enforce formatting IMHO

Reply:
> let them skip.

I'm afraid but if you allow that, won't[ this same thing](https://github.com/appium/appium-xcuitest-driver/pull/1972#discussion_r1320289041) happen the next time the file is updated? I would say this would not be a good experience for a reviewer and reviewee as well. I think the problem is that there is no consistency in the application of the format rule.
In my opinion, if you want to align the format, there should be a guardrail to prevent developers from going off the rail as much as possible, as this repository is checking the linter rule in CI. Or if you're not particular about the format and think it is fine without the rule, I would suggest removing the husky settings. WDYT?

Reply:
> Or if you're not particular about the format and think it is fine without the rule, I would suggest removing the prettier setting from husky.

Fine with me

Reply:
Okay! Let me close this and have another PR for it.

Reply:
PTAL https://github.com/appium/appium-xcuitest-driver/pull/1974

Reply:
you may still keep this PR. I am totally ok with changes made by prettier unless they are mixed with anything else ;)

Reply:
I see, then let me reopen and remove CI part